students/rmart300/trigram/trigram.py

```diff
@@ -0,0 +1,45 @@
+
+trigram_dict={}
+
+def read_book(book_file):
+
+    book = open(book_file, 'r'):
+    digram = ''
+    for line in book:
+        line.strip()
```

Comment:
this line will have no effect since strings are immutable, if you do want this to take effect you can do:
`words = line.strip().split()`